src/dsp_tools/utils/xmlupload/stash/stash_models.py

```diff
@@ -49,3 +46,60 @@ def make(tups: list[tuple[XMLResource, StandoffStashItem]]) -> StandoffStash | N
             else:
                 res_2_stash_items[xmlres.id].append(stash_item)
         return StandoffStash(res_2_stash_items, res_2_xmlres)
+
+
+@dataclass(frozen=True)
+class LinkValueStashItem:
+    """Holds information about a single stashed link value."""
+
+    res_id: str
+    res_type: str
+    prop_name: str
+    target_id: str
```

Comment:
The permissions are still missing right?

Reply:
yes, I would like to keep that in a separate task, as it would require quite a bit of work to understand how the permissions work/should work